src/common/crypto.cpp

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "aos/common/crypto.hpp"
+
+namespace aos {
+namespace crypto {
+
+/***********************************************************************************************************************
+ * Public
+ **********************************************************************************************************************/
+
+//
+// RSAPublicKey implementation
+//
+
+RSAPublicKey::RSAPublicKey(const Array<uint8_t>& n, const Array<uint8_t>& e)
```

Comment:
Remove cpp file and put implementation directly into header file. These are mostly const methods. Also we initilly designed this lib to be header only. It is not always possible. But we try to keep it in this format.

Reply:
done